website/templates/project/modal_generate_private_link.mako

```diff
@@ -41,15 +41,21 @@
 
 
                     <div class="row">
-
                         <div class="col-md-6" >
                             <div class="list-overflow">
                             <input type="checkbox" checked disabled />
+
                             <span data-bind="text:title"></span> (current component
                                 <span data-bind="if: isPublic">, public</span>)
+
                             <div data-bind="foreach:nodes">
                                 <div data-bind="style:{'marginLeft': margin}">
+                                    <!-- ko if: ($parent.nodesToChange().indexOf($data.parent_id) !== -1 ||  $data.parent_id === $parent.id())-->
```

Comment:
Can this check move to the view model? Prefer to keep complex logic out of templates, plus easier to write tests for code in view models than in templates.